icu4c/source/data/cldr-icu-readme.txt

```diff
@@ -140,8 +140,7 @@
 # 1a. Java and ant variables, adjust for your system
 
 export JAVA_HOME=`/usr/libexec/java_home`
-mkdir -p /tmp/cldrdtd
-export ANT_OPTS="-Xmx4096m -DCLDR_DTD_CACHE=/tmp/cldrdtd"
+export ANT_OPTS="-Xmx4096m -DCLDR_DTD_CACHE=~/.cldrdtd"
 
```

Comment:
Why are you removing the mkdir here (for whatever dir is used)? The cache directory is not created if it does not exist. If you don't explicitly create it then the cache will not be populated and yo will get blocked.

Reply:
I moved the mkdir down to step 1d

Reply:
Oh, I see, I did not have enough context when I made the provious comment, I thought the mkdir was much further down. I just looked at the whole file. Seems fine.